src/dataIntegration/transcriptomics/MBA/MBA.m

```diff
@@ -0,0 +1,73 @@
+function tissueModel = MBA(model, expressionRxns, threshold_medium, threshold_high, tol, core)
+% Uses the MBA algorithm (`Jerby et al., 2010`) to extract a context

```

Comment:
As with fastcore: I would suggest to handle the setting of the different groups outside of the algorithm (higher flexibility, users can choose what they want). 
i.e. change the header to 
```'
function tissueModel = MBA(model,medium_set, high_set, tol)
```
This is closer to the description of the algorithm in the paper (my opinion).